tests/test_commandline

```diff
@@ -174,4 +174,66 @@ fi
 
 echo "Test 3 passed"
 
+# Test 4: --tpmstate backend=file parameters test
```

Comment:
I think you'll need to add a call to `cleanup` before the `echo` above to clean up the TPMDIR created in line 107.

Reply:
Correct. I'll add `cleanup` just after `echo "Test 3 passed"` as Test 1 does.